citylots_bench_test.go

```diff
@@ -0,0 +1,60 @@
+package quamina
```

Comment:
Assuming this is just extraction to a new file… any logical changes to the benchmark?

Reply:
It's the same benchmark code, the main difference (and this why I kept the original) is that I couldn't go benchmarks to emit events per second, which is an important metric.

Anyway, I am going to remove this benchmark from this PR and add it (with CI as discussed in the past) here - https://github.com/timbray/quamina/pull/126 (not yet ready for review)